trajectory_tracker_msgs/msg/TrajectoryTrackerStatus.msg

```diff
@@ -7,4 +7,4 @@ Header header
 float32 distance_remains
 float32 angle_remains
 int32 status
-
+time path_stamp
```

Comment:
Will we also want to add `path_frame_id` and `path_seq`?
`seq` may be useful to know that the trajectory_tracker received a new path message.
Storing as `path_header` might be an another option.

Reply:
Done